src/jrd/ExtEngineManager.cpp

```diff
@@ -888,6 +888,7 @@ ExtEngineManager::Trigger::Trigger(thread_db* tdbb, MemoryPool& pool, ExtEngineM
 ExtEngineManager::Trigger::~Trigger()
 {
 	// hvlad: shouldn't we call trigger->dispose() here ?
```

Comment:
The comment looks confusing after the change, please remove it and I'll merge this PR.

Reply:
Done. Squished this fix to first commit.